### PR TITLE
Es6ify

### PR DESCRIPTION
**Pull request by noseglid**
Use classes and features of es6
Also use eslint rather than JSCS and JSHint

### CODE REVIEW

File: spec/build-error-match-spec.js [hunk 1]
```diff
@@ -1,24 +1,24 @@
-'use strict';
+'use babel';
 
-var fs = require('fs-extra');
-var temp = require('temp');
-var specHelpers = require('atom-build-spec-helpers');
+const fs = require('fs-extra');
+const temp = require('temp');
+const specHelpers = require('atom-build-spec-helpers');
```
Comment (kepler0) [inline spec/build-error-match-spec.js]: Why not use `import` over `require`?

Comment (noseglid) [inline spec/build-error-match-spec.js]: Of course I should. I'll fix that!

File: spec/build-keymap-spec.js [hunk 2]
```diff
@@ -1,17 +1,16 @@
 'use babel';
-'use strict';
 
-var fs = require('fs-extra');
-var temp = require('temp');
-var specHelpers = require('atom-build-spec-helpers');
+const fs = require('fs-extra');
+const temp = require('temp');
+const specHelpers = require('atom-build-spec-helpers');
```
Comment (kepler0) [inline spec/build-keymap-spec.js]: `import`?

File: spec/build-spec.js [hunk 3]
```diff
@@ -1,25 +1,24 @@
 'use babel';
-'use strict';
 
-var fs = require('fs-extra');
-var temp = require('temp');
-var specHelpers = require('atom-build-spec-helpers');
+const fs = require('fs-extra');
+const temp = require('temp');
+const specHelpers = require('atom-build-spec-helpers');
```
Comment (kepler0) [inline spec/build-spec.js]: `import`?

File: spec/build-targets-spec.js [hunk 4]
```diff
@@ -1,18 +1,17 @@
...
-'use strict';
 
-var _ = require('lodash');
-var fs = require('fs-extra');
-var temp = require('temp');
-var specHelpers = require('atom-build-spec-helpers');
+const _ = require('lodash');
+const fs = require('fs-extra');
+const temp = require('temp');
+const specHelpers = require('atom-build-spec-helpers');
```
Comment (kepler0) [inline spec/build-targets-spec.js]: `import`?

File: spec/build-view-spec.js [hunk 5]
```diff
@@ -1,18 +1,16 @@
 'use babel';
-'use strict';
 
-var fs = require('fs-extra');
-var temp = require('temp');
-var specHelpers = require('atom-build-spec-helpers');
+const fs = require('fs-extra');
+const temp = require('temp');
+const specHelpers = require('atom-build-spec-helpers');
```
Comment (kepler0) [inline spec/build-view-spec.js]: `import`?

File: spec/build-visible-spec.js [hunk 6]
```diff
@@ -1,18 +1,16 @@
 'use babel';
-'use strict';
 
-var fs = require('fs-extra');
-var temp = require('temp');
-var specHelpers = require('atom-build-spec-helpers');
+const fs = require('fs-extra');
+const temp = require('temp');
+const specHelpers = require('atom-build-spec-helpers');
```
Comment (kepler0) [inline spec/build-visible-spec.js]: `import`?

File: spec/build-confirm-spec.js [hunk 7]
```diff
@@ -1,18 +1,17 @@
...
-'use strict';
 
-var _ = require('lodash');
-var fs = require('fs-extra');
-var temp = require('temp');
-var specHelpers = require('atom-build-spec-helpers');
+const _ = require('lodash');
+const fs = require('fs-extra');
+const temp = require('temp');
+const specHelpers = require('atom-build-spec-helpers');
```
Comment (kepler0) [inline spec/build-confirm-spec.js]: `import`?

**Comment by kepler0** [comment]
Other than the `import` over `const ... require()` stuff it looks great. :+1:

**Pull request merged**